bc_obps/service/facility_service.py

```diff
@@ -248,3 +249,21 @@ def generate_bcghg_id(cls, user_guid: UUID, facility_id: UUID) -> BcGreenhouseGa
         facility.save(update_fields=['bcghg_id'])
 
         return facility.bcghg_id
+
+    # brianna check facility transfer is atomic
```

Comment:
```suggestion

```

Reply:
It isn't: https://github.com/orgs/bcgov/projects/123/views/20?pane=issue&itemId=96053742&issue=bcgov%7Ccas-registration%7C2776

Reply:
I also would have thought we didn't need to wrap child methods, but I was getting some weird behaviour while testing. I'll try again later and update/delete that card